metricbeat/module/elasticsearch/pending_tasks/data.go

```diff
@@ -0,0 +1,39 @@
+package pending_tasks
+
+import (
+	"encoding/json"
+
+	"github.com/elastic/beats/libbeat/common"
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstriface"
+)
+
+var (
+	schema = s.Schema{
+		"insert_order":     c.Int("insert_order"),
+		"priority":         c.Str("priority"),
+		"source":           c.Str("source"),
+		"time_in_queue.ms": c.Int("time_in_queue_millis"),
```

Comment:
This should be as following to make sure internally it's already an object. But outcome on the ES side is identical. We can do this in a follow up PR.

```
"time_in_queue": s.Object{
  "ms": c.Int("time_in_queue_millis"),
}
```